packages/form-js-viewer/test/spec/render/components/form-fields/Text.spec.js

```diff
@@ -123,6 +125,104 @@ Some _em_ **strong** [text](#text) \`code\`.
   });
 
 
+  it('should render markdown (expression)', function() {
+
+    // given
+    const content = '#foo';
+
+    const { container } = createText({
+      data: {
+        content
+      },
+      field: {
+        text: '=foo',
+        type: 'text'
+      },
+      evaluateExpression: () => content
+    });
+
+    // then
+    const formField = container.querySelector('.fjs-form-field');
+
+    expect(formField).to.exist;
+    expect(formField.innerHTML).to.eql('<div class="markup"><div xmlns="http://www.w3.org/1999/xhtml"><h1>foo</h1></div></div>');
+  });
+
+
+  it('should render markdown (complex expression)', function() {
+
+    // given
+    const content = [ '#foo', '###bar' ];
+
+    const { container } = createText({
+      data: {
+        content
+      },
+      field: {
+        text: '=foo',
+        type: 'text'
+      },
+      evaluateExpression: () => content
+    });
+
+    // then
+    const formField = container.querySelector('.fjs-form-field');
+
+    expect(formField).to.exist;
+    expect(formField.innerHTML).to.eql('<div class="markup"><div xmlns="http://www.w3.org/1999/xhtml"><h1>foo,###bar</h1></div></div>');
+  });
+
+
+  describe('disabled', function() {
```

Comment:
On the Operate team we created a rule to avoid nested describes so it's easier to read and it's helps with keeping the tests isolated.

If you and @Skaiir like the idea we could consider doing this here too

Reply:
I'm not sure actually. I'm in favor of nested describe blocks, to structure them logically. I guess this just personal taste, no strong feelings though.